tpgtools/resource.go

```diff
@@ -166,7 +166,7 @@ func (l Link) Markdown() string {
 }
 
 func (r *Resource) fillLinksFromExtensionsMap(m map[string]interface{}) {
-	ref, ok := m["x-dcl-ref"].(map[string]interface{})
+	ref, ok := m["x-dcl-ref"].(map[interface{}]interface{})
```

Comment:
With the previous line, ref is always an empty map and ok is always false and that's not the case with the new line. I suspect this has something to do with how the yaml is being read, but I'm not sure why the value is a `map[interface{}]interface{}` rather than a `map[string]interface{}`.